<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeamsWindow</class>
 <widget class="QMainWindow" name="TeamsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="teams_win_header_widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="teams_team_label">
         <property name="text">
          <string>Team:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="teams_team_name_text">
         <property name="text">
          <string>team name</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="teams_win_body_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeWidget" name="team_persons_tree_widget">
         <property name="showDropIndicator" stdset="0">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>200</number>
         </attribute>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Username</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Org ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>is Team Admin</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="teams_win_person_btns_widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="teams_add_person_BTN">
         <property name="text">
          <string>Add new person</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="teams_change_role_BTN">
         <property name="text">
          <string>change role</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="teams_remove_person_BTN">
         <property name="text">
          <string>remove person</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="teams_win_footer_widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="team_seach_by_label">
         <property name="text">
          <string>Search by:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="search_by_name_radio">
         <property name="text">
          <string>Name</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="search_by_username_radio">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="search_by_org_id_radio">
         <property name="text">
          <string>Org ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="search_line_edit">
         <property name="placeholderText">
          <string>Search ...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="members_checkbox">
         <property name="text">
          <string>Members</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Admins_checkbox">
         <property name="text">
          <string>Admins</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
